Main/ClientApp/src/Actions/CollectionActions.ts

```diff
@@ -49,6 +49,40 @@ export async function loadSharedCollections() {
   });
 }
 
+export async function loadContributorsSharedCollections() {
+  let contributorsSharedCollections = await SharedCollectionsApi.getContributorsSharedCollections();
+  Dispatcher.dispatch({
+    actionType: ActionTypes.LOAD_CONTRIBUTORS_SHARED_COLLECTIONS,
+    payload: { contributorsSharedCollections: contributorsSharedCollections },
+  });
+}
+
 export async function shareCollection(shareData: SharedCollectionData) {
-  await SharedCollectionsApi.shareCollection(shareData);
+  let success = await SharedCollectionsApi.shareCollection(shareData);
+  if (success) {
+    loadCollections();
+    loadSharedCollections();
+    loadContributorsSharedCollections();
+  }
+}
+
+export async function updateSharedCollection(updateData: SharedCollectionData) {
+  let success = await SharedCollectionsApi.updateSharedCollection(updateData);
+  if (success) {
+    loadSharedCollections();
+    loadContributorsSharedCollections();
+  } else console.error("Could not update shared collection");
+}
+
+export async function deleteContributorSharedCollection(
```

Comment:
please don't save on names by removing the words, code should be readable. "deleteContributorOfSharedCollection" or "deleteSharedCollectionContributor" would be a nicer fit.